functions-and-scope.js

```diff
@@ -82,3 +167,22 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+// ANTWOORD Opdracht 3b
+
+function compareForHighestGrade(grades) {
+
+
+    let highestFoundGrade = 0;
```

Comment:
Het is handiger om als default waarde de eerste waarde uit de array te pakken in plaats van `0`. Stel je voor dat je `const grades = [-2, -4, -1];` als waarde hebt, dan zou `highestFoundGrade` uiteindelijk de waarde `0` hebben terwijl die waarde helemaal niet eens in de array voorkomt 😉